Tests/MLXTests/TransformTests.swift

```diff
@@ -240,35 +240,32 @@ class TransformTests: XCTestCase {
         XCTAssertFalse(allClose(c2a, c2b).item())
     }
 
-    func testCompilePerformance() {
-        // this is the code from compilation.md
-
-        // disabling until we pick up the fix for https://github.com/ml-explore/mlx/issues/31
-        return
-
-        func measure(_ f: (MLXArray) -> MLXArray, _ x: MLXArray) {
-            // warm up
-            for _ in 0 ..< 10 {
-                eval(f(x))
-            }
-
-            let start = Date.timeIntervalSinceReferenceDate
-            let iterations = 100
-            for _ in 0 ..< iterations {
-                eval(f(x))
-            }
-            let end = Date.timeIntervalSinceReferenceDate
-
-            let timePerIteration = 1000.0 * (end - start) / Double(iterations)
-
-            print("Time per iteration \(timePerIteration.formatted()) ms")
-        }
-
-        let x = MLXRandom.uniform(0 ..< 1, [32, 1000, 4096])
-
-        measure(gelu, x)
-        measure(compile(gelu), x)
-    }
+    // disabling until we pick up the fix for https://github.com/ml-explore/mlx/issues/31
+//    func testCompilePerformance() {
+//        // this is the code from compilation.md
+//        func measure(_ f: (MLXArray) -> MLXArray, _ x: MLXArray) {
+//            // warm up
+//            for _ in 0 ..< 10 {
+//                eval(f(x))
+//            }
+//
+//            let start = Date.timeIntervalSinceReferenceDate
+//            let iterations = 100
+//            for _ in 0 ..< iterations {
+//                eval(f(x))
+//            }
+//            let end = Date.timeIntervalSinceReferenceDate
+//
+//            let timePerIteration = 1000.0 * (end - start) / Double(iterations)
+//
+//            print("Time per iteration \(timePerIteration.formatted()) ms")
+//        }
+//
+//        let x = MLXRandom.uniform(0 ..< 1, [32, 1000, 4096])
+//
+//        measure(gelu, x)
+//        measure(compile(gelu), x)
+//    }
```

Comment:
This is addressed in https://github.com/ml-explore/mlx-swift/pull/6 -- perhaps we should omit this here as it will conflict.

Reply:
Got it. Done Could you help trigger CI for me?

Reply:
I will trigger the CI build after #22 is merged